sklearn/cluster/_optics.py

```diff
@@ -601,27 +601,27 @@ def cluster_optics_xi(*, reachability, predecessor, ordering, min_samples,
         Expressed as an absolute number or a fraction of the number of samples
         (rounded to be at least 2).
 
-    min_cluster_size : int > 1 or float between 0 and 1 (default=None)
+    min_cluster_size : int > 1 or float between 0 and 1, default=None
         Minimum number of samples in an OPTICS cluster, expressed as an
         absolute number or a fraction of the number of samples (rounded to be
         at least 2). If ``None``, the value of ``min_samples`` is used instead.
 
-    xi : float, between 0 and 1, optional (default=0.05)
+    xi : float, between 0 and 1, default=0.05
```

Comment:
Same here (remove the comma before `between`).

Reply:
ok